en/05_Contributing/05_Coding_Conventions/01_PHP_Coding_Conventions.md

```diff
@@ -136,6 +136,133 @@ class MyClass extends Class
 }
 ```
 
+## Code style
+
+- Prioritise the human readability of code. Avoid "clever" code that aims to reduce the number of lines at the expense of readbability. This is particularly important when writing Javascript.
+- Avoid writing any new "magic" code as it is very hard to follow.
+- Within reason, favour larger files that have everything contained in them rather than multiple small files that require developers to jump around multiple files to follow a single code path.
+
+## Inline comments
+- If there is a non-obvious reason for a code operation, include verbose inline comments that explain *why* we are doing this operation. Remember that while something may be obvious to you now, it won't be obvious to someone else in 6 months.
+- Inline comments that explain *what* the code is doing are generally fine as it can make code easier to follow and is particularly useful for larger methods, though it may be better to simply split larger methods in to several private methods.
```

Comment:
```suggestion
- Inline comments that explain *what* the code is doing are generally fine as it can make the code easier to follow and could be particularly useful for larger methods, though it may be better to simply split larger methods into several private methods.
```

Reply:
Done